packages/versions/package.json

```diff
@@ -10,8 +10,8 @@
     "fuels-versions": "dist/bin.global.js"
   },
   "exports": {
-    ".": "./src/index.ts",
-    "./cli": "./src/cli.ts"
+    ".": "./dist/index.js",
+    "./cli": "./dist/cli.js"
```

Comment:
Are you still using the `@fuel-ts/versions` to fetch the versions?

If not, perhaps this workaround can be removed before merging.

Also because of #827